### ISSUE TITLE
Vispy integration ?

### ISSUE THREAD

**Issue by stuaxo** [swistakm/pyimgui#43]
Vispy could be a good one to integrate with, especially since it can work in jupyter notebooks.

http://vispy.org/

**Comment by swistakm** [comment]
Yeah, that would be great. It seems `vispy` is based on OpenGL so it may be possible to reuse at least parts of the existing code for that.

**Comment by stuaxo** [comment]
Ugh ... so many different bindings to OpenGL - I wish there was a way to make them cooperate - but this is O/T.

**Comment by lubieowoce** [comment]
I'm interested in using vispy, it'd be very useful for a project I'm working on. 

Unfortunately, from a cursory look it seems that vispy prefers to run the whole app rather than just offer some visualisation components. The model suggested by their API seems to be "a Vispy app that calls a gui backend to render everything" rather than "a gui app that calls Vispy to render some plots and handles the rest on its own", which is kind of disappointing... I hope I'm wrong about that! I'll look into it more soon.
